.github/workflows/validate-markdown-links.yml

```diff
@@ -0,0 +1,18 @@
+name: Validate Markdown Links for PRs
+  # This tests links using a *local* copy of the site *without* building it
+  # That means links need to be accurate before Jekyll builds and the page is deployed
+
+on:
+  pull_request
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        uses: lycheeverse/lychee-action@v1.8.0
```

Comment:
👍 very nice, haven't seen this one before. definitely adding it to some other projects now

Reply:
So this is great for scanning documents locally - but it does have some issues.  That `--base .` is absolutely necessary or it just ignores links that start with `/`.

It also isn't good at crawling all of a live sites links (vs. a folder full of files) so I'm setting up the site scanner using a completely different tool called `muffet`.  PR pending soon...

Reply:
Issue no 1265 in the lychee repo